platformio.ini

```diff
@@ -9,7 +9,11 @@
 ; http://docs.platformio.org/page/projectconf.html
 
 [env:d1_mini]
-platform = espressif8266
+platform = espressif8266@2.0.4
 board = d1_mini
 framework = arduino
 lib_deps = https://github.com/marvinroger/homie-esp8266.git
+
+; Configute your COM-Port here
+upload_port = COM7
+monitor_speed = 115200 ; as set in SenseoWifi.cpp in L318
```

Comment:
Good call on those changes!
Is the COM port defined in the ini file needed? What happens if we leave it out? I'D like to avoid invalid default configuration
You can remove the second comment, it's pretty clear

Reply:
I am not sure how PlatformIO behaves without a given COM port.
I'll remove this line and also the 2nd comment.